microbench/src/main/java/io/netty/handler/codec/http2/HpackHeader.java

```diff
@@ -75,6 +75,12 @@
             }
         } else {
             r.nextBytes(bytes);
+
+            // Remove leading ':' as this is not allowed
+            if (removeLeadingColon && bytes[0] == ':') {
```

Comment:
s/if/while/

Reply:
@ejona86 ? we add it from the ASCII array so there can no ':' anymore

Reply:
I see... In that case, why are we creating binary header keys? Values can have `obs-text` and in general a wider range of characters. But the name is a `token` and `token` is:

```
     token          = 1*tchar

     tchar          = "!" / "#" / "$" / "%" / "&" / "'" / "*"
                    / "+" / "-" / "." / "^" / "_" / "`" / "|" / "~"
                    / DIGIT / ALPHA
                    ; any VCHAR, except delimiters
```

We should just always use ALPHABET for the keys.